libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -282,6 +282,9 @@ static DNNReturnType load_tf_model(TFModel *tf_model, const char *model_filename
         TF_SetConfig(sess_opts, sess_config, sess_config_length,tf_model->status);
```

Comment:
for line 271 and 272, do they conflict with function ff_dnn_free_model_tf?  If so, we might delete line 271 and 272.